source/_integrations/rpi_gpio.markdown

```diff
@@ -197,3 +197,93 @@ switch:
     ports:
       17: Speaker Relay
 ```
+## Light
+
+The `rpi_gpio` light platform allows you to use a Raspberry Pi to control your lights.
+
+It uses two pins on the Raspberry Pi.
+
+- The `light_button_pin` will detect if the light button pushed
+- the `relay_pin` will turn on the light relay.
+
+
+### Configuration
+
+To enable Raspberry Pi Light in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: rpi_gpio
+    lights:
+      - name: my light
+        relay_pin: 18
+        light_button_pin: 17
```

Comment:
We no longer accept YAML configuration for platform configurations.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>

These platforms need to be converted to an integration configuration first.